src/index.html

```diff
@@ -17,16 +17,44 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      method="get"
+      class="search-bar search-bar--big"
+      data-qa="big"
+    >
+      <label
+        class="search-bar__label--big"
+        for="icon-big"
+      >
+        <input
+          class="search-bar__input search-bar__input--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+          id="icon-big"
+        />
+      </label>
+    </form>
 
```

Comment:
There should be an empty line between the two form elements to improve readability, as per the checklist requirement to add empty lines between multiline sibling blocks of HTML.